tensorflow/lite/kernels/strided_slice.cc

```diff
@@ -59,8 +59,7 @@ struct StridedSliceContext {
   int dims;
 };
 
-// This Op only supports 1-4D cases and since we use the reference 4D
-// implementation, the 1-3D tensors are mapped to 4D.
+// This Op only supports 1-4D cases
 const int kMaxDim = 4;
```

Comment:
I'm confused, we still require 4D? So what exactly is the improvement with this change?

Reply:
@jdduke : Thanks for reviewing the changes.
As i mentioned in case of slice Op changes also.
The Strided Slice Op also closely bound with structure
(For example: https://github.com/tensorflow/tensorflow/blob/3a094e6d1c927967438c0b263db0bf42d16813ae/tensorflow/lite/kernels/internal/types.h#L1041) 
which has limitation of 4. And those structures are used multiple places. As the effort is more for removing all those places.
I think it would be better, i take that as part of different PR.
But to do that, first current PR should get merged.
For simplicity in terms of review, i have segregated in that manner.
Please let me know your opinion. Thanks!